lib/framebuffer.cc

```diff
@@ -112,6 +112,56 @@ class DirectRowAddressSetter : public RowAddressSetter {
   int last_row_;
 };
 
+class SM5266RowAddressSetter : public RowAddressSetter {
```

Comment:
Could you write a brief description of how the setting of bits works here ?
Looks like the lower 3 bits are clocked in to din_, while bit 4 and 5 are set directly ?

(it sounds weird that they do this, while they have perfectly good three direct bits available that they now use to clock in exactly three bits; possibly a result for how they internally do the multiplexing ?)